dodola/tests/test_services.py

```diff
@@ -82,3 +82,32 @@ def test_bias_correct_basic_call():
         fakestorage.storage[output_key]["fakevariable"].squeeze(drop=True).values[-5:],
         tail_vals,
     )
+
+
+def test_build_weights(tmpdir):
+    """Test that services.build_weights produces a weights file"""
+    # Output to tmp dir so we cleanup & don't clobber existing files...
+    weightsfile = tmpdir.join("a_file_path_weights.nc")
+
+    # Make fake input data.
+    lat_n = 30
+    lon_n = 60
+    ds_in = xr.Dataset(
+        {"fakevariable": (["lon", "lat"], np.ones(shape=(lon_n, lat_n)))},
+        coords={
+            "lon": (["lon"], np.linspace(-180, 179, lon_n)),
+            "lat": (["lat"], np.linspace(-90, 90, lat_n)),
+        },
+    )
+
+    fakestorage = FakeRepository(
+        {
+            "a_file_path": ds_in,
+        }
+    )
+
+    build_weights(
+        "a_file_path", method="bilinear", storage=fakestorage, outpath=weightsfile
```

Comment:
Can we test both `bilinear` and `conservative` method options by adding a `@pytest.mark.parametrize("method", ['bilinear', 'conservative']` line above this test function and a method argument?

Reply:
Good idea. Done in 9df03a1.